@xen-orchestra/vmware-explorer/esxi.mjs

```diff
@@ -357,4 +358,36 @@ export default class Esxi extends EventEmitter {
   powerOn(vmId) {
     return this.#exec('PowerOnVM_Task', { _this: vmId })
   }
+  async fetchProperty(type, id, propertyName) {
+    // the fetch method does not seems to be exposed by the wsdl
+    // inspired by the pyvmomi implementation ( StubAdapterAccessorImpl.py / InvokeAccessor)
+    const res = await fetch(`https://${this.#host}/sdk`, {
```

Comment:
Construct the URL properly instead of concatenating: 
```suggestion
    const url = new URL('https://localhost/sdk')
    url.host = this.#host
    const res = await fetch(url, {
```